debian/Dockerfile.systemd.partial

```diff
@@ -0,0 +1,24 @@
+# Install Systemd
```

Comment:
Why don't we use a `Dockerfile.tpl` and `Dockerfile.nosystemd.tpl` just like it was in the systemd directory?